_docs/utils/css-variables.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: docs
+title: CSS variables
+description: CSS variables
+group: utils
+---
+
+CSS variables are available with the format: `--env-{variable}`
+
+Only works in browsers that support <a class="doc-link" href="https://caniuse.com/#feat=css-variables" target="_blank">CSS variables.</a>
```

Comment:
It is a bad practice to link to cross-origin destinations using `target="_blank"` without including either `rel="noopener"` or `rel="noreferrer"`. For more on this see this explanation, https://web.dev/external-anchors-use-rel-noopener/ 

```suggestion
Only works in browsers that support <a class="doc-link" href="https://caniuse.com/#feat=css-variables" target="_blank" rel="noopener">CSS variables.</a>
```